### PR TITLE
Tidy up hs

### PR DESCRIPTION
**Pull request by lc512k**
cc @keirog 

relates to https://github.com/Financial-Times/n-teaser-collection/pull/12

### CODE REVIEW

File: views/partials/body-video.html [hunk 1]
```diff
@@ -32,14 +32,14 @@ <h1 class="video__placeholder__headline article-headline">
...
-								<h4>Up Next</h4>
+								<h2><span aria-hidden="true">Up Next</span><span class="n-util-visually-hidden">Related videos</span></h2>
 								<ol class="js-placeholder-up-next"></ol>
 							</div>
 						</div>
 					{{else}}
 						<div class="video__error">
 							<h1>This video cannot be displayed</h1>
-							<p>We just updated our video hub and some videos might take a while till they are accessible. Please come back later or watch the clips below instead.</p>
+							<p>We just updated our video hub and some videos might take a while before they can be accessed. Please come back later or watch the clips below instead.</p>
```
Comment (keirog) [inline views/partials/body-video.html]: ❤️

File: client/components/special-report/_main.scss [hunk 2]
```diff
@@ -1,5 +1,12 @@
 .stream-card {
 }
+
+.stream-card__title {
+    margin: 1em 0;
+    font-size: 1.17em;
+    font-weight: bold;
```
Comment (keirog) [inline client/components/special-report/_main.scss]: In lieu of a usable set of typography rules at the moment, 🙈  👍  (I think something is in the pipeline though)

Comment (lc512k) [inline client/components/special-report/_main.scss]: yeah, this is very 😬 

before it was just ~~ab~~using the default h3 styles; it didn't have any styles of its own

File: views/partials/body-video.html (hunk 1 above)
Comment (lc512k) [inline views/partials/body-video.html]: yeah, happy accident. Might be a bit longer of a while till they _are_ accessible ;)

**Pull request merged**